libs/shared/src/consts/handlebar-helpers/getTemplateVariables.ts

```diff
@@ -0,0 +1,91 @@
+import { TemplateVariableTypeEnum } from '../../entities/message-template';
+import { HandlebarHelpersEnum } from './handlebarHelpers';
+
+export interface IMustacheVariable {
+  type: TemplateVariableTypeEnum;
+  name: string;
+  defaultValue?: string | boolean;
+  required?: boolean;
+}
+
+export function getTemplateVariables(bod: any[]): IMustacheVariable[] {
```

Comment:
I am wondering if shared folder is the best place for this. So far, we don't really have functions in the shared folder. I wouldn't want to create a precedence that we are not interested in. 
This function will be used frequently, in web&api. 
Also, testing wise, it would make more sense to test this whole variables calculations here, but - again - we don't have testing in shared and maybe it makes more sense not to.  For now, it is being tested in api -> doesn't feel clean enough. 

Let me know what you think ❤️ As I'm not sure I like this like that